lib/improver/tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -232,16 +236,29 @@ class Test_check_compatible_time_points(Test_weighted_blend):
 
     """Test the time point compatibility checking function."""
 
-    def test_forecast_reference_time_exception(self):
-        """Test that a ValueError is raised if the coordinate to be blended
-        is forecast_reference_time and the points on the time coordinate are
-        not equal."""
+    def test_unmatched_validity_time_exception(self):
+        """Test that a ValueError is raised if the validity time of the
+        slices over the blending coordinate differ. We should only be blending
+        data at equivalent times unless we are triangular time blending."""
+        self.cube.remove_coord('time')
+        self.cube.coord('forecast_reference_time').rename('time')
```

Comment:
Would be good to update the error message below / above (in the code) to reflect the new logic.

Reply:
Done.